tests/unit/test_particle_swarm_optimizer.py

```diff
@@ -1,5 +1,12 @@
+import sys
+import os
+
 import unittest
 from torch import randint, device as torch_device, Tensor
+
```

Comment:
This is probably not required. The tests run from the root folder without this change. i.e

```
python -m unittest tests/unit/test*
```